gradle/dependencies.gradle

```diff
@@ -34,7 +34,7 @@ def build = [
         errorProneTestHelpers: "com.google.errorprone:error_prone_test_helpers:${versions.errorProne}",
         nullAway             : 'com.uber.nullaway:nullaway:0.8.0',
         gradlePlugins        : [
-                android   : '7.0.0-alpha01',
```

Comment:
Don't do alpha versions

Reply:
About this downgrade of the gradle pluging. The problem is that the version 7.0.0-alpha01  is no more available.
And to upgrade to 7.0.0  will be necessary to upgrade the gradle version from 6 to 7. If i do this, will be necessary to rewrite a good part of the gradle fliles to the new syntax.
So i downgraded to the last stable version before 7.0.0.